pkg/network/protocols/ebpf_types_linux_test.go

```diff
@@ -0,0 +1,3 @@
+// Code generated by genpost.go; DO NOT EDIT.
+
+package protocols
```

Comment:
that's an empty file. Why do we need it?

Reply:
I think this is because the test generation isn't preventing generation of the test file when there are no `struct` types in the paired file. It is a minor bug.

Reply:
Fixed